test/test_log_statement.py

```diff
@@ -9,6 +9,7 @@ def test_no_args(self):
     def test_one_arg(self):
         self.assertEqual('console.log("hello")', main.main('print("hello")', 'py', 'js'))
         self.assertEqual('print(5)', main.main('console.log(5)', 'js', 'py'))
+        self.assertEqual('echo 8', main.main('console.log(8)', 'js', 'bash'))
```

Comment:
This is the problem with only doing these big integration tests. Need to now ask the question if we should translate btwn all the combinations of language pairs in order to test. Not sure there is a good answer and I think integration tests are important. @Danish21 we talked about this during our 1:1. Do you have any advice?

Reply:
I agree that our testing framework is not perfect. Also, Michael and I decided it may not be worth it to continue to have bash as an input language, so we will definitely need gast_to_code tests for bash

Reply:
I think it would be valuable to also do some lang <-> gast tests if those don't exist already. 
Tests are pretty cheap so why not also have a bunch of lang -> lang tests as well.

Reply:
We aren't going to do bash as an input language anymore but I think those are valuable for gast to code